semantic_segmentation_server:
  camera:
    color_topic : /camera/color/image_raw
  model:
    model_name : "deeplabv3plus"
    encoder_name : "resnet34"
    encoder_weights : "imagenet"
    in_channels : 3
    classes : 2
    model_path: $(find semantic_segmentation_ros)/assets/model/deeplabv3plus_171_mIOU=0.9686.pt